src/cli/main.py

```diff
@@ -27,23 +33,37 @@ def getting_started(name: str):
 @app.command()
 def build_skill_assessment(name: str, skills: str = ''):
     "Builds skill assessment using standards passed."
-    print(f"Building Standards in /standards folder {name}")
+    print(f"Building {name} assesment in /assessment/user folder")
     # TODO standards index
     # Open Standards
-    file_and_path = Path("tests/data") / f"{name}.json"
-    skill_matrix_overview_path = "cli/templates/skill_matrix_overview.md"
+    if environment == 'production':
+        file_and_path_string = "standards/" + name + ".json"
+    else:
+        file_and_path_string = "src/cli/tests/data/" + name + ".json"
+
+    # file_and_path = Path("tests/data") / f"{name}.json"
+    file_and_path = Path(file_and_path_string)
+    skill_matrix_overview_path = "cli/templates/skill_matrix_overview.md" 
+
 
     # Convert to markdown
-    with open(file_and_path, "r", encoding="utf-8") as file:
-        standards_json = json.load(file)
+    try:
+        with open(file_and_path, "r", encoding="utf-8") as file:
+            standards_json = json.load(file)
+    except:
+        print('No file '+file_and_path_string)
+        return
     # TODO: Discovery functions
     # print(standards_json)
     skills_assment_matrix = helpers.flatten_categories(standards_json)
     skills_assment_matrix = helpers.mixin_skill_assessment_details(
         skills_assment_matrix
     )
-    with open(skill_matrix_overview_path, "r") as overview_file:
-        skill_matrix_overview = overview_file.read()
+    try:
+        with open(skill_matrix_overview_path, "r") as overview_file:
+            skill_matrix_overview = overview_file.read()
+    except:
```

Comment:
Can you add a specific exception instead of a catch all?

 I've never used the pkg_resources library before, what is that used for?

Reply:
pkg_resources allows you to pull in static files for packages.